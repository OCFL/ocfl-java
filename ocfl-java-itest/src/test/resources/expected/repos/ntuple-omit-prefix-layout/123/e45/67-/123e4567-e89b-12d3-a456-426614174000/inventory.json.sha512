2665465a1338131b58fd51684fb4791242672b922888164ae1ed27e9b54442ad8e40ffeddad682a3cdc88366db11438c13978ee2a4a2494938ae99bbfffc1933  inventory.json
