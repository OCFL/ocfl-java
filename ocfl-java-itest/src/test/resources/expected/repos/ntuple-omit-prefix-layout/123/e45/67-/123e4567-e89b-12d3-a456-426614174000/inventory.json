{
  "id" : "urn:uuid:123e4567-e89b-12d3-a456-426614174000",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v1",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "0e3e75234abc68f4378a86b3f4b32a198ba301845b0cd6e50106e874345700cc6663a86c1ea125dc5e92be17c98f9a0f85ca9d5f595db2012f7cc3571945c123" : [ "v1/content/f1.txt" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2022-01-25T10:52:24.500340725-06:00",
      "message" : "ntuple omit prefix test",
      "user" : {
        "name" : "Example",
        "address" : "mailto:test@example.com"
      },
      "state" : {
        "0e3e75234abc68f4378a86b3f4b32a198ba301845b0cd6e50106e874345700cc6663a86c1ea125dc5e92be17c98f9a0f85ca9d5f595db2012f7cc3571945c123" : [ "f1.txt" ]
      }
    }
  }
}